src/helpers/connectorsNext.ts

```diff
@@ -17,6 +18,13 @@ export const availableConnectors = isInArgentMobileAppBrowser()
       }),
     ]
   : [
+      new Argent({
+        mobile: {
+          url: typeof window !== "undefined" ? window.location.href : "",
+          dappName: "Example dapp",
+          chainId: CHAIN_ID,
+        },
+      }),
       new InjectedConnector({ options: { id: "argentX" } }),
```

Comment:
this should be removed right?

Reply:
Why, we could always use testing of all other connectors?

Reply:
right, the 1 button doesn't replace the single connectors 
they can still be used as standalone

👍

Reply:
Yup, I'd like to remove them, but that would be a breaking change, so I'd rather keep that major bump for refactoring. Then, we can decide whether to remove them. That way we're giving folks some time to test the new one.